README.md

```diff
@@ -109,7 +109,7 @@ otel:
     filter:
       include:
         match_type: regexp
-        resource_attributes:
+        record_attributes:
```

Comment:
This was changed in the previous PR. Reverting back, because the README is for the current stable release, which does still work with `record_attributes`.